okhttp/src/test/java/okhttp3/internal/http2/HttpOverHttp2Test.java

```diff
@@ -112,12 +112,12 @@
     return asList(Protocol.H2_PRIOR_KNOWLEDGE, Protocol.HTTP_2);
   }
 
-  private PlatformRule platform = new PlatformRule();
+  private final PlatformRule platform = new PlatformRule();
+  private final OkHttpClientTestRule clientTestRule = new OkHttpClientTestRule();
```

Comment:
should we combine the two rule classes? Might make things simpler from callsites?

Reply:
Yep.  PlatformRule has a terrible name, so let's kill that and just make it a feature of OkHttpClientTestRule